icinga2/command.conf

```diff
@@ -0,0 +1,33 @@
+object CheckCommand "os_release" {
+	import "plugin-check-command"
+
+	command = [ PluginDir + "/check_os_release" ]
+
+	arguments = {
```

Comment:
Could you please also add the new --eolIgnore and --releaseIgnore switches from #3?